tests/gold_tests/remap/base.replay.yaml

```diff
@@ -0,0 +1,49 @@
+meta:
+  version: '1.0'
+sessions:
+- protocol:
+  - name: http
+    version: 1
+  transactions:
+  - client-request:
+      headers:
+        fields:
+        - - Content-Length
+          - 0
+        - - uuid
+          - get
+        - - X-Request
+          - get
+      method: GET
+      url: /test/ip_allow/test_get
+      version: '1.1'
+    proxy-response:
+      status: 200
+    server-response:
+      headers:
+        fields:
+        - - Content-Length
+          - 20
+      reason: OK
+      status: 200
```

Comment:
Let's keep the headers below the reason, status, method, etc. nodes.

Reply:
You mean the order of fields, right? I changed them to follow other replay.yaml.